content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,109 @@
 ---
-title: "Fundamental Wellbeing"
+title: "A Scientific Approach to Awakening and Fundamental Wellbeing: The Work of Dr. Jeffrey Martin"
 created: 2023-04-21
-authors: 
+authors:
   - rufuspollock
-  - jefferymartin
+description: Fundamental Wellbeing is the experience of persistent, higher states of conscious, also known as awakening, enlightenment and other terms. This page is an introduction to the concept.
+image: /assets/images/fundamental-wellbeing-dr-jeffery-martin.jpg
 ---
-## What is Fundamental Wellbeing
+Dr. Jeffery Martin is a researcher known for his work on Fundamental Wellbeing, which focuses on understanding and cultivating higher states of consciousness and well-being in individuals. He has conducted extensive studies exploring various altered states of consciousness, such as enlightenment, mystical experiences, and non-symbolic consciousness.
 
-Fundamental Wellbeing is a termed coined (or made popular) by [Jeffery Martin](https://lifeitself.org/notes/jeffery-martin). In more academic settings he tends to refer to it as Persistent Non-Symbolic Experience (Martin credits the "non-symbolic" terminology to [abc/@cook-greuter](https://lifeitself.org/abc/@cook-greuter)).
```

Comment:
We may want to keep some of this more detailed background especially the references to wilber, cook creuter etc but move to an appendix or a section down the page like "More about Fundamental Wellbeing" (and its relation to other similar terms)